src/Appwrite/Extend/Exception.php

```diff
@@ -66,6 +66,7 @@ class Exception extends \Exception
     public const USER_SESSION_ALREADY_EXISTS       = 'user_session_already_exists';
     public const USER_NOT_FOUND                    = 'user_not_found';
     public const USER_PASSWORD_RECENTLY_USED       = 'password_recently_used';
+    public const USER_PASSWORD_IN_DICTIONARY       = 'password_in_dictionary';
```

Comment:
I cant see where we use this new excepiton

Reply:
removed